src/mount/mount-tool.c

```diff
@@ -1130,6 +1304,17 @@ int main(int argc, char* argv[]) {
                 goto finish;
         }
 
+        r = bus_connect_transport_systemd(arg_transport, arg_host, arg_user, &bus);
+        if (r < 0) {
+                log_error_errno(r, "Failed to create bus connection: %m");
+                goto finish;
+        }
+
+        if (arg_action == ACTION_UMOUNT) {
+                r = action_umount(bus, argc, argv);
+                goto finish;
+        }
```

Comment:
hmm, so the --help text needs updating too. the first line should be duplicated for the ---umount command:

```
systemd-mount [OPTIONS...] WHAT [WHERE]
systemd-mount [OPTIONS...] WHAT|WHERE…